clang/lib/Driver/Driver.cpp

```diff
@@ -4720,6 +4914,18 @@ InputInfo Driver::BuildJobsForActionNoCache(
                         C.addTempFile(C.getArgs().MakeArgString(TmpFileName),
                                       types::TY_Tempfilelist);
         CurI = InputInfo(types::TY_Tempfilelist, TmpFile, TmpFile);
+      } else if (EffectiveTriple.getSubArch() ==
+                                         llvm::Triple::SPIRSubArch_fpga &&
```

Comment:
Indentation is slightly off I think

Reply:
Another 'let me fit this in 80 columns' format weirdness.  I'll change the ordering so it fits better.